src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -869,12 +869,27 @@ impl EventProcessor {
         // Set the timestamp.
         self.target.xconn.set_timestamp(xev.time as xproto::Timestamp);
 
+        // A key input implies having the focus
         let window = match self.active_window {
             Some(window) => window,
-            None => return,
+            None => *self.active_window.insert(xev.window as xproto::Window),
         };
 
         let window_id = mkwid(window);
+        let focused = self.with_window(window, |window| {
```

Comment:
Should add a comment explaining why this is needed and tell that others also do something like that.

Reply:
Ok, will do.